Formula/msgpack-cxx.rb

```diff
@@ -0,0 +1,57 @@
+class MsgpackCxx < Formula
+  desc "MessagePack implementation for C++ / msgpack.org[C++]"
+  homepage "https://msgpack.org/"
+  url "https://github.com/msgpack/msgpack-c/releases/download/cpp-4.0.1/msgpack-cxx-4.0.1.tar.gz"
+  sha256 "acba76ebc3ff9af2fd2649f319b25cd2387c66f75ae74a7a652fe3386e45d498"
+  license "BSL-1.0"
+  head "https://github.com/msgpack/msgpack-c.git", branch: "cpp_master"
+
+  depends_on "cmake" => :build
+  depends_on "boost"
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+
+    # fix include conflicts between msgpack and msgpack-cxx
+    include.install include/"msgpack" => "msgpack-cxx"
+    include.install include/"msgpack.hpp" => "msgpack-cxx.hpp"
+    inreplace include/"msgpack-cxx.hpp", "\"msgpack/", "\"msgpack-cxx/"
```

Comment:
Oops. Looks like renaming didn't work. Let's drop the renaming, and add the conflict in a follow-up PR so we can keep the `msgpack` bottle for High Sierra.

Reply:
Yup, don't add `conflicts_with` for now. Let's add it right after this is merged.

Reply:
I see your poiint.